src/WebGL.ts

```diff
@@ -2979,8 +2988,8 @@ export class WebGLRenderingContext {
 	@inline getActiveUniformBlockName(program: WebGLProgram, uniformBlockIndex: GLuint): DOMString {
 		return getActiveUniformBlockName(this.gl_id, program, uniformBlockIndex);
 	}
-	@inline uniformBlockBinding(program: WebGLProgram, uniformBlockIndex: GLuint, uniformBlockBinding: GLuint): void {
-		uniformBlockBinding(this.gl_id, program, uniformBlockIndex, uniformBlockBinding);
+	@inline uniformBlockBinding(program: WebGLProgram, uniformBlockIndex: GLuint, _uniformBlockBinding: GLuint): void {
+		uniformBlockBinding(this.gl_id, program, uniformBlockIndex, _uniformBlockBinding);
```

Comment:
name collision avoided (otherwise inside the scope of the function `uniformBlockBinding` is a number, not a function, therefore we can not call it. Maybe @inline also hid this error, but TS/VSCode shows it).